extensions/amp-sidebar/1.0/toolbar.js

```diff
@@ -103,36 +100,19 @@ export class Toolbar {
     this.toolbarClone_ = this.toolbarDOMElement_.cloneNode(true);
     const targetId = this.toolbarDOMElement_.getAttribute('target');
     // Set the target element to the toolbar clone if it exists.
-    const targetElement =
-    this.win_.document.getElementById(`${targetId}_toolbar`) ||
-    this.win_.document.getElementById(targetId);
-    this.targetElement_ = targetElement || this.createTargetElement_();
-    this.targetElement_.appendChild(this.toolbarClone_);
-    toggle(this.targetElement_, false);
-
-    // Check if the target element was created by us, or already inserted by the user
-    if (!this.targetElement_.parentElement) {
+    const targetElement = this.win_.document.getElementById(targetId);
+    if (targetElement) {
+      this.targetElement_ = targetElement;
       this.toolbarClone_.classList.add(TOOLBAR_ELEMENT_CLASS);
-      const fragment = this.win_
-        .document.createDocumentFragment();
-      fragment.appendChild(this.targetElement_);
-      this.body_.appendChild(fragment);
+      this.targetElement_.appendChild(this.toolbarClone_);
+      toggle(this.targetElement_, false);
+    } else {
+      throw new
```

Comment:
The creation of toolbars i wrapped in a try/catch that throws user Errors if there is an error building toolbars, see: https://github.com/ampproject/amphtml/blob/master/extensions/amp-sidebar/1.0/amp-sidebar.js#L117

Would you like to remove the try/catch, and simply do the user error here?

Also, I trust that sidebar should continue to be functional. I feel that it is better to have some content missing than all of it. UX wise, It does not hinder the core functionality of sidebar for this to break, and would still be a positive experience for the rest of sidebar elements.